src/apps/mmfinance/cronos/MMfinance.auto-mmf.contract-position-fetcher.ts

```diff
@@ -0,0 +1,54 @@
+import { Inject } from '@nestjs/common';
+import { BigNumber } from 'ethers';
+
+import { APP_TOOLKIT, IAppToolkit } from '~app-toolkit/app-toolkit.interface';
+import { Register } from '~app-toolkit/decorators';
+import { RewardRateUnit } from '~app-toolkit/helpers/master-chef/master-chef.contract-position-helper';
+import { PositionFetcher } from '~position/position-fetcher.interface';
+import { ContractPosition } from '~position/position.interface';
+import { Network } from '~types/network.interface';
+
+import { MmfinanceContractFactory, MmfinanceMmfChef } from '../contracts';
+import { MMFINANCE_DEFINITION } from '../mmfinance.definition';
+
+const appId = MMFINANCE_DEFINITION.id;
+const groupId = MMFINANCE_DEFINITION.groups.autoMmf.id;
+const network = Network.CRONOS_MAINNET;
+
+@Register.ContractPositionFetcher({ appId, groupId, network })
+export class CronosChainMmfinanceAutoCakeContractPositionFetcher implements PositionFetcher<ContractPosition> {
+  constructor(
+    @Inject(APP_TOOLKIT) private readonly appToolkit: IAppToolkit,
+    @Inject(MmfinanceContractFactory)
+    private readonly contractFactory: MmfinanceContractFactory,
+  ) { }
```

Comment:
Please ensure your IDE is configured to use the linting settings in our repository. This should be `{}` not `{ }`. You can alternatively run the linting scripts in `package.json`.